UnitTests/BraintreeThreeDSecureTests/BTPaymentFlowDriver+ThreeDSecure_Tests.swift

```diff
@@ -76,6 +78,41 @@ class BTPaymentFlowDriver_ThreeDSecure_Tests: XCTestCase {
         waitForExpectations(timeout: 3, handler: nil)
     }
 
+    func testPerformThreeDSecureLookup_whenAddCardChallengeNotRequested_sendsAddCardFalse() {
```

Comment:
```suggestion
    func testPerformThreeDSecureLookup_whenCardAddChallengeNotRequested_sendsAddCardFalse() {
```

Reply:
Do we also need a test for if the cardAddChallenge was set to `.requested`? 🤔

Reply:
The `.requested` case is covered in `testPerformThreeDSecureLookup_sendsAllParameters`

Reply:
Ah gotcha - good call